### PR TITLE
Update delegation figures

### PR DESCRIPTION
**Pull request by crypto-matto**
- [x] Hide unbonding balance & status entry point when no on-going unbonding delegations
- [x] Update delegations formula
- Total Balance = Unstaked balance + Staked Balance + Unbonding Balance in transition + Reward Balance
- Total Rewards = Claimed rewards + Reward Balance

螢幕截圖 2021-09-17 下午5 09 25
螢幕截圖 2021-09-17 下午5 09 39

### CODE REVIEW

File: src/pages/staking/staking.tsx [hunk 1]
```diff
@@ -1606,132 +1586,19 @@ const StakingPage = () => {
...
-                          {userAsset?.symbol}
-                        </div>
-                      )}
-                      <div className="fiat">
-                        {userAsset && marketData && marketData.price
-                          ? `${SUPPORTED_CURRENCY.get(marketData.currency)?.symbol}${numeral(
-                              getAssetRewardsBalancePrice(userAsset, marketData),
-                            ).format('0,0.00')} ${marketData?.currency}
-                          `
-                          : ''}
```
Comment (crypto-3dd) [inline src/pages/staking/staking.tsx:1654]: BTW I wonder why this section is being removed completely ?

**Comment by cdc-Hitesh** [comment]
One question:

1. Does the unbonding CRO balance auto-hide when ZERO?

File: src/pages/staking/staking.tsx (hunk 1 above)
Comment (crypto-matto) [reply src/pages/staking/staking.tsx:1654]: The `Total Rewards` modal is moved to Home Page instead.

**Comment by crypto-matto** [comment]
> One question:
> 
> 1. Does the unbonding CRO balance auto-hide when ZERO?

Nope it's not. Would you suggest to?

**Review by crypto-3dd — APPROVED** [review]
LGTM

**Pull request merged**